steps/expression.go

```diff
@@ -111,40 +128,115 @@ func (p *Expression) makeFunctions() map[string]govaluate.ExpressionFunction {
 			return float64(time.Now().Unix()), nil
 		}),
 		"num": p.makeStringFunction("num", 0, func(_ *bitflow.Sample, _ ...string) (interface{}, error) {
-			return p.num, nil
+			return float64(p.num), nil
 		}),
 		"str": func(arguments ...interface{}) (interface{}, error) {
 			if len(arguments) == 1 {
 				return fmt.Sprintf("%v", arguments[0]), nil
 			}
 			return nil, fmt.Errorf("str() needs 1 parameter, but received: %v", printParamStrings(arguments))
 		},
+		"strToFloat": func(arguments ...interface{}) (interface{}, error) {
```

Comment:
Recommend to rename to `parse_float` to maintain a consistent naming scheme of these functions which are named with underscores instead of camel case.

Reply:
Done